code/vel/python/lab09_ver1.py

```diff
@@ -0,0 +1,41 @@
+"""
+Lab 09: Peaks and Valleys
+"""
+data = [1, 2, 3, 4, 5, 6, 7, 6, 5, 4, 5, 6, 7, 8, 9, 8, 7, 6, 7, 8, 9]
+
+def find_peaks():
+    peaks = []
+    for i in range(1, len(data)-1): # range(start, stop, step) start = 1
+        if data[i -1] < data[i] and data[i + 1] < data[i]:
+            peaks.append(i)
+    return peaks
+
+#IndexError index out of range
```

Comment:
In the future, try to clean up these kinds of troubleshooting/coding comments before submitting.